pages/GLAM3D/disseminate.md

```diff
@@ -285,7 +285,7 @@ Local Contexts’ [Traditional Knowledge labels](http://localcontexts.org/tk-lab
      The Creative Commons Public Domain Mark may be appropriate if your institution has positively identified a work to be in the public domain. _See Public Domain Mark 1.0_, Creative Commons, [https://creativecommons.org/ publicdomain/mark/1.0/](https://creativecommons.org/publicdomain/mark/1.0/), last accessed April 16, 2020.
 
 [^60]:
-     Rightsstatements.org provides 12 standardized rights statements, each with a unique URI and accompanying text, to help with this process.
```

Comment:
Michael, I read the blog post. I don't include Rightstatements as best practice for OA. It allows institutions to continue to not responsibly carry out their obligations to rights clear content. Rightsstatemetns are also too varied and not legally operative. My only recommended use for Rightstatements is internal processing in a DAM system to flag asset statuses within the organization. Rightstatements should not be use for public consumption of content. In suggesting this OpenGLAM and their partners are milseading institutions and users. They are not contributing to best practice.

Reply:
That's fine, but the reference to rights statement was already in the piece.  Is your objection to having them there at all or to the additional cite? To the extent that they are already in the document, I found this site to be helpful context in understanding how people are using the rights statements.

Reply:
I would not cite beyond the link already in the document. I wouldn't want to point to source that doesn't offer information I wouldn't recommend to others.

Reply:
Can you flesh out your concerns about rightstatement a bit more?  I view it as a tool (like CC) that can - but does not necessarily - help clarify the open status of a work.  Just like CC has NC or SA licenses, RS has ambiguous options that do not necessarily advance openness. However, they also have standard ways to make it clear to users that the works are freely available.

Beyond that, I don't see the additional link as an endorsement of RS.  RS has always been somewhat confusing to me.  I found the information in the link to be very helpful to contextualize what RS is attempting to do and explain how it exists in relation to CC.  If we live in a world where RS exists and is part of the conversation people are having around open access, I don't understand the benefit of not providing people  information about what it even is.

Reply:
After a further conversation we decided that best practice should be to research work status/restrictions and to convey them in as non-legal a manner as possible. As a result we decided not to add additional pointers to RS at this time.